test/include/nntrainer_test_util.h

```diff
@@ -212,6 +213,20 @@ const std::string config_str2 = "[Network]"
     }                                                        \
   } while (0)
 
+
+/**
+ * @brief test tensor is equal
+ * @param[in] Tensor A, tensor to be compared
+ * @param[in] Tensor B, tensor to be compared
+ */
+void test_tensor_eq(nntrainer::Tensor const &A, nntrainer::Tensor const &B);
```

Comment:
This isn't important but you can consider overloading `operator ==` in `tensor class` for checking equality.

Reply:
True. I'll make a issue for that.